docker/Dockerfile

```
@@ -35,6 +35,9 @@ RUN install-package \
     ninja \
     cmake \
     meson \
+    nodejs \
```

Use the LTS ([fermium is v14](https://wiki.archlinux.org/index.php/Node.js_#Installation), the current LTS):

```suggestion
    nodejs-lts-fermium \
```